source/includes/fundamentals/explain-verbosity.rst

```diff
@@ -0,0 +1,23 @@
+The following table shows the verbosity levels for the explain
+operation and their intended use cases:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 40 60
+
+   * - Verbosity Level
+     - Use Case
+
+   * - ALL_PLANS_EXECUTIONS
+     - You want to know which plan MongoDB will choose to run your query.
+
+   * - EXECUTION_STATS
+     - You want to know if your query is performing well.
+
+   * - QUERY_PLANNER
+     - You have a problem with your query and you want as much information
+     as possible to diagnose why it might be slow.
```

Comment:
```suggestion
     - You have a problem with your query and you want as much information
     as possible to diagnose the issue.
```

Suggestion: This sentence sounded slightly off the first time I read it, and I think the issue for me was that "a problem" sounds general "but might be slow" sounds specific. I think either "might be slow" should be made more general (like "the issue") or "a problem" should be made more specific (like "Your app is slow").